src/views/v2/components/loader.php

```diff
@@ -9,15 +9,17 @@
  *
  * @link http://m.tri.be/1aiy
  *
- * @version 4.9.10
- *
+ * @version TBD
  */
 ?>
```

Comment:
* :no_entry_sign: There must be exactly one blank line after the file comment
* :no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: c662e397c07ee2031214e56a5b02f8eb</sub>